docs/nodes/validator-node.md

```diff
@@ -72,11 +72,11 @@ go version go1.17.2 linux/amd64
 
 ## Deploying The Celestia App
 
-This section describes part 1 of Celestia Bridge Node setup: running a Celestia
-App daemon with an internal Celestia Core node.
+This section describes part 1 of Celestia Validator Bridge Node setup:
```

Comment:
What's a validator bridge node?

Reply:
It was a weird naming thing (trying to reference both running a Validator and a Bridge node) but fixed it.